dbt_subprojects/daily_spellbook/models/near/ft_transfer_calls/near_ft_transfer_calls.sql

```diff
@@ -0,0 +1,76 @@
+{{
+  config(
+    schema = 'near_ft_transfer_calls',
+    alias = 'ft_transfer_calls',
+    materialized = 'table',
```

Comment:
transfers are a good candidate for incremental models, to lessen the load on compute infra each run. plz refer to EVM transfers, such as [this](https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/tokens/models/transfers_and_balances/ethereum/tokens_ethereum_base_transfers.sql#L1-L11), for example of how to set up config block on incremental.